### DISCUSSION TITLE
ERROR [EmailProcessor] Error processing send-email job. Reason: Greeting never received

### DISCUSSION THREAD

**Discussion by vnetesov** [docmost/docmost#277] [Help / Questions] [answered]
<img width="349" alt="изображение" src="https://github.com/user-attachments/assets/29554f74-1a97-41b5-b6d4-867e251f3d4c">

What could be the error with the mail settings?

**Comment by Philipinho** [comment, marked as answer]
Add `SMTP_SECURE=true`.

**Reply by vnetesov** [reply]
Everything worked out, thanks!!
